README.md

```diff
@@ -1,6 +1,6 @@
-# Elab Report Downloader
+# eLab Report Downloader (DAA)
```

Comment:
This is a general elab tool, because the inside format for all the labs is same and you only need to change the link to download reports from different labs.
Do you want to add something in those terms?

Reply:
Alright, yeah, it'll probably work for all formats. I guess I made it DAA for my reference.

Reply:
Sure, can you make the changes and i will merge the PR.